ansible/roles/packages/tasks/install-flatcar.yaml

```diff
@@ -0,0 +1,120 @@
+---
+- include: url.yaml
+# must include crictl-url.yml after installing containerd,
+# as the cri-containerd tarball also includes crictl.
+- include: crictl-url.yaml
+
+- name: create containerd systemd directory
+  file:
+    path: "/etc/systemd/system/containerd.service.d/"
+    state: directory
+
+- name: create containerd directory
+  file:
+    path: "/etc/containerd/"
+    state: directory
+
+- name: copy default configuration to remote
+  template:
+    src: "config.toml.tmpl"
+    dest: "/etc/containerd/config.toml"
+
+
+- name: Fail if pre-installed containerd version does not match for Flatcar
+  shell: containerd --version | cut -d\  -f 3
+  register: command_result
+  failed_when: "containerd_version not in command_result.stdout"
+
+- name: Create a directory if it does not exist
+  file:
+    path: "{{ sysusr_prefix }}/bin"
+    state: directory
+    mode: 0755
+
+- name: download containerd
+  get_url:
+    url: "{{ containerd_url }}"
+    checksum: "sha256:{{ containerd_sha256 }}"
+    dest: /tmp/containerd.tar.gz
+    mode: 0600
+
+- name: unpack containerd
+  unarchive:
+    remote_src: True
+    src: /tmp/containerd.tar.gz
+    dest: /opt
+    extra_opts:
+      - --no-overwrite-dir
+      - --strip-components=2
+      - --wildcards
+      - '*containerd-shim-runc-v*'
+
+- name: Create symlinks for containerd
+  file:
+    src: /usr/bin/{{ item }}
```

Comment:
flatcar instances typically are only writable under /opt and /etc

Reply:
good point. think it it runs through probably because links are already there

Reply:
i think this can be handled in the seperate flatcar PR. This just took over what was already in image builder but in separated files.